src/plugins/telemetryTable/collections/TableRowCollection.js

```diff
@@ -117,6 +142,20 @@ define(['lodash', 'EventEmitter'], function (_, EventEmitter) {
       return sortedRows;
     }
 
+    insertRows(rowsToAdd, addToBeginning) {
```

Comment:
Why are we re-inserting the updated rows? The table row and cell Vue components are referencing a shared instance of a`TelemetryTableRow` object. All we need to do is trigger a view update in `table-row.vue` when the datum changes in-place. We could emit an `updated` event maybe?

Reply:
I renamed the method to `insertOrUpdateRows`
Since the `row` is a prop to `table-row.vue`, the view updates automatically when the row is updated in the `TableRowCollection` [here](https://github.com/nasa/openmct/pull/6694/files#diff-8c09e0bdafd88bead579797b6f91ffd217675e1a2e1e8480a908aadb5fbd253aR131)
So we don't need an `updated` event, right?

Reply:
My previous comment is resolved, it wasn't relevant.